pkg/sdk/go/domains_test.go

```diff
@@ -0,0 +1,1151 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package sdk_test
+
+import (
```

Comment:
Add test for `ListDomainUsers`

Reply:
Covered in users tests, is refactored by #2195